README.md

```diff
@@ -30,185 +30,210 @@ Specifications:
 
+1. Ensure you have mysql and boost installed.
+    
     ```shell
-    export PKG_CONFIG_PATH=/usr/local/qt5/5.5/clang_64/lib/pkgconfig
-    export PATH=/usr/local/qt5/5.5/clang_64/bin:$PATH
-    export QT_CFLAGS="-I/usr/local/qt5/5.5/clang_64/lib/QtWebKitWidgets.framework/Versions/5/Headers -I/usr/local/qt5/5.5/clang_64/lib/QtWebView.framework/Versions/5/Headers -I/usr/local/qt5/5.5/clang_64/lib/QtDBus.framework/Versions/5/Headers -I/usr/local/qt5/5.5/clang_64/lib/QtWidgets.framework/Versions/5/Headers -I/usr/local/qt5/5.5/clang_64/lib/QtWebKit.framework/Versions/5/Headers -I/usr/local/qt5/5.5/clang_64/lib/QtNetwork.framework/Versions/5/Headers -I/usr/local/qt5/5.5/clang_64/lib/QtGui.framework/Versions/5/Headers -I/usr/local/qt5/5.5/clang_64/lib/QtCore.framework/Versions/5/Headers -I. -I/usr/local/qt5/5.5/clang_64/mkspecs/macx-clang -F/usr/local/qt5/5.5/clang_64/lib"
-    export QT_LIBS="-F/usr/local/qt5/5.5/clang_64/lib -framework QtWidgets -framework QtGui -framework QtCore -framework DiskArbitration -framework IOKit -framework OpenGL -framework AGL -framework QtNetwork -framework QtWebKit -framework QtWebKitWidgets -framework QtDBus -framework QtWebView"
+    brew install mysql boost
     ```
 
-5. Install the other required items:
+2. Ensure you have python 2.7 installed and in your path (OS X comes with this by default)
 
-    `brew install protobuf boost miniupnpc openssl qrencode berkeley-db4 automake`
+    ```shell
+    python --version
+    ```
 
-6. Download the wallet source and build:
+3. Export the required environment variables
 
     ```shell
-    git clone https://github.com/vergecurrency/verge
-    cd verge
-    ./autogen.sh
-    ./configure --with-gui=qt5
-    make -j4
+    export VERGE_PLATFORM='mac'
+    export CXX=clang++
+    export CC=clang
     ```
 
-Building the Mac installer (.dmg) file
------
-Run `make deploy`
+4. Run your build commands
 
-If you are building the .dmg (by running 'mac deploy') you may need to run these commands if you get an error regarding mysql:
+    ```shell
+    ./building/common.sh
+    ./building/mac/requirements.sh
+    ./building/mac/build.sh
+    ```
 
-```shell
-brew install mysql
-cd /usr/local/qt5/5.4/clang_64/plugins/sqldrivers
-otool -L libqsqlmysql.dylib
-```
+5. Grab a :coffee: and wait it out
 
-Note: This may be pointing to an version of mysql that you do not have installed (like mysql55) - Alternatively, you may be able to remove the library from the sqldrivers folder.
+6. Create the `.dmg` file
 
-```
-install_name_tool -change /opt/local/lib/mysql55/mysql/libmysqlclient.18.dylib /usr/local/Cellar/mysql/5.7.12/lib/libmysqlclient.20.dylib libqsqlmysql.dylib
-```
+    ```shell
+    ./building/mac/dist.sh
+    ```
 
-Trying to build .dmg on 10.8? You will need to run this:
+### Windows Wallet
 
-```shell
-export CFLAGS=-Qunused-arguments
-export CPPFLAGS=-Qunused-arguments
-sudo -E easy_install appscript
-```
+TODO. Take a look as [building/windows](./building/windows).
```

Comment:
@justinvforvendetta I don't know if you have any updated windows instructions kicking around but would you be able to add them here?

Reply:
yes, ill get on that. its changed a bit since we dont need qtwebkit or qtwebkitwidgets now. TODO noted.